### ISSUE TITLE
Can't go to other dimensions.

### ISSUE THREAD

**Issue by TwitchyM** [McJty/RFToolsDimensions#200]
Issue description: Stepping on the teleporter puts the "teleporting" message in the chat box but i never get ported, i just stay in place.

Steps to reproduce: Try to teleport to any dimension. 

Versions:
* Minecraft: 1.12.2
* Forge: 1.12.2
* CompatLayer (only if on Minecraft 1.10 or 1.11):
* McJtyLib: 1.12
* RFTools: 1.12
* RFTools Dimensions: 1.12.1

Possibly incompatible mods: Optifine, chameleon, cxlibrary, sonar-core.

Relevant logs, if any:

**Comment by josephcsible** [comment]
Please give correct values for versions. What you posted is obviously not correct. Also, upload your log file; I'm pretty sure that it will have useful information.

**Comment by TwitchyM** [comment]
Ok for one those are the exact versions I'm playing with most have not been updated to 1.12.2 but 'obviously' still work because I'm playing right now. Two, I figured out the issue while fiddling around in creative it was a mod that changes ore generation that I deleted after I created the dimensions but before i tried to port to it. The dimension wasn't usable because it was made with the modded ore rather than the vanilla. Sorry to have wasted your time, I should have messed with it before going for outside help.
